message/FieldMessageSerializer.cpp

```diff
@@ -0,0 +1,67 @@
+#include <cmath>
+#include <sstream>
+#include "FieldMessageSerializer.h"
+
+std::string FieldMessageSerializer::Serialize(const FieldMessage &message) {
+    std::ostringstream oss;
+
+    auto messageSize = message.GetMessageSize();
+    auto bitMask = message.GetBitmask();
+    oss.write((char *)&messageSize, sizeof(messageSize));
```

Comment:
`(char *)` is a secret `reinterpret_cast`((

Reply:
What's wrong with casting? I don't know any other ways to serialize something in binary form

Reply:
It used to be fashionable to write casts [like this](https://github.com/jart/cosmopolitan/blob/master/libc/intrin/memcmp.c)
Behind this deprecated cast is reinterpret_cast, which it is customary (like other casts) to write explicitly

Reply:
> It used to be fashionable to write casts [like this](https://github.com/jart/cosmopolitan/blob/master/libc/intrin/memcmp.c) Behind this deprecated cast is `reinterpret_cast`, which it is customary (like other casts: `const_cast`, `static_cast`, `dynamic_cast`) to write explicitly

Actually this cast is not deprecated in C++. C-style casts are more flexible. But I agree that in C++ it is recommended to use a explicit type casting syntax.